client/blocks/reader-combined-card/index.jsx

```diff
@@ -120,7 +120,7 @@ class ReaderCombinedCard extends React.Component {
 				<ul className="reader-combined-card__post-list">
 					{ posts.map( post => (
 						<ReaderCombinedCardPost
-							key={ `post-${ post.ID }` }
+							key={ `post-${ postKey.feedId || postKey.blogId }-${ postKey.postId }` }
```

Comment:
postId doesn't exist on a combined card postKey. postIds does and it's an array. You'll need to match up the index with the index passed to the map callback...